.storybook/stories/components/Navigation/Logo.stories.js

```diff
@@ -0,0 +1,15 @@
+import { Logo } from 'components/Header/Logo'
```

Comment:
Temporarily removing it, since this `Logo` component will be used in the `Header`, thus it will be added as a part of  the `Header`story in the future implementation. Thank you!